tests/test_spatialstats.py

```diff
@@ -21,16 +23,16 @@
 
 PLOT = False
 
-def load_ref_and_diff() -> tuple[gu.georaster.Raster, gu.georaster.Raster, np.ndarray]:
+def load_ref_and_diff() -> tuple[Raster, Raster, np.ndarray, Vector]:
     """Load example files to try coregistration methods with."""
 
-    reference_raster = gu.georaster.Raster(examples.get_path("longyearbyen_ref_dem"))
-    glacier_mask = gu.geovector.Vector(examples.get_path("longyearbyen_glacier_outlines"))
+    reference_raster = Raster(examples.get_path("longyearbyen_ref_dem"))
+    glacier_mask = Vector(examples.get_path("longyearbyen_glacier_outlines"))
 
-    ddem = gu.georaster.Raster(examples.get_path('longyearbyen_ddem'))
+    ddem = Raster(examples.get_path('longyearbyen_ddem'))
     mask = glacier_mask.create_mask(ddem)
 
-    return reference_raster, ddem, mask
+    return reference_raster, ddem, mask, glacier_mask
```

Comment:
`mask` is created but never used...

Reply:
It is used in some, `test_estimate_model_spatial_correlation_and_infer_from_stable` for instance.
I removed it from the examples where it was passed at the first line, but not used later.

Reply:
Sorry I misread, I thought `mask` was not returned anymore, only `glacier_mask`. All good then!